Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -5781,39 +9571,43 @@ def ip_list_member_list_command(client: Client, args: dict[str, Any]) -> Command
     Returns:
         CommandResults: outputs, readable outputs and raw response for XSOAR.
     """
-    group_name = args["group_name"]
-    member_id = args.get("member_id")
-    response = client.ip_list_member_list_request(
-        group_name=group_name, member_id=member_id
-    )
-    parsed_data, pagination_message, formatted_response = list_response_handler(
-        client, response, client.parser.parse_ip_list_member, args, member_id
+    name = args.get("name")
+    response = client.url_access_rule_group_list_request(name=name)
+    parsed_data, pagination_message, _ = list_response_handler(
+        client=client,
+        response=response,
+        data_parser=client.parser.parse_url_access_rule_group,
+        args=args,
+        sub_object_id=name,
+        sub_object_key=client.parser.sub_object_id_key,
     )
-    outputs = {"group_name": group_name, "Members": parsed_data}
     headers = client.parser.create_output_headers(
-        client.version, ["id", "type", "ip"], ["severity", "trigger_policy"], []
+        client.version,
+        ["id", "action", "host", "severity", "count"],
+        [],
```

Comment:
Why not create the create_output_headers method with [] as default values and then not send empty arrays as inputs?